pkg/term/term_linux_test.go

```diff
@@ -3,29 +3,20 @@
 package term
 
 import (
-	"flag"
 	"io/ioutil"
 	"os"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-var rootEnabled bool
-
-func init() {
-	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
The intent was to only run some tests if the flag was set. I think the correct behaviour is to optimistically run the tests that can be run when the user is correct.